docs/sources/mobile-app/push-notifications/index.md

```diff
@@ -83,3 +84,18 @@ You can also enable or disable Do Not Disturb override for **important** notific
 On-call shift notifications are sent to announce upcoming shifts, roughly ~15 minutes in advance.
 
 To enable or disable on-call shift notifications, use the **On-call shift notifications** section in the **Push notifications** settings.
+
+### Shift swap notifications
+
+Shift swap notifications are generated when a [shift swap request][shift-swaps] is created,
+informing all users in the on-call schedule (except the initiator) about potential schedule changes.
+
+Shift swap notifications are sent one month ahead of the requested shift swap, during users' working hours
```

Comment:
```suggestion
Shift swap notifications are sent one month ahead of the requested shift swap, or shortly after creation in case the shift swap start time is less then a month away, but always during users' working hours
```